include/SANDKalmanFilter.h

```diff
@@ -0,0 +1,217 @@
+#ifndef SANDKALMANFILTER_H
+#define SANDKALMANFILTER_H
+
+#include "TGeoManager.h"
+#include "TMatrixD.h"
+
+#include "SANDTrackerClusterCollection.h"
+#include "SANDKFTrack.h"
+#include "SANDTrackerUtils.h"
+
+struct SParticleInfo {
+  int charge;
+  double mass;
+  int pdg_code;
+  int id;
+
+  TVector3 pos;
+  TVector3 mom;
+};
+
```

Comment:
Instead of repeating `SANDKF... `for each type used in this file, this would be a great place to declare some namespaces to improve scoping and readability.
`sand::kf::Manager` `sand::kf::Track` `sand::kf::StateVector`




https://google.github.io/styleguide/cppguide.html#Namespaces